src/mmw/js/src/app.js

```diff
@@ -15,17 +15,44 @@ var App = new Marionette.Application({
 
         // This view is intentionally not attached to any region.
         this._mapView = new views.MapView({
-            model: this.map
+            model: this.map,
+            mapId: 'map'
         });
 
         this.rootView = new views.RootView();
         this.user = new userModels.UserModel({});
 
-        var header = new views.HeaderView({
+        this.header = new views.HeaderView({
```

Comment:
Outside the scope of this PR, but this should probably get moved to [here](https://github.com/WikiWatershed/model-my-watershed/blob/develop/src/mmw/js/src/draw/controllers.js#L16) and should not be part of the application.